pandas/core/frame.py

```diff
@@ -374,7 +375,7 @@ class DataFrame(NDFrame):
     """
 
     @property
-    def _constructor(self):
+    def _constructor(self) -> Type['DataFrame']:
```

Comment:
I think need to use double quotes or will fail black

Reply:
ok, I will update the PR tomorrow. Totally forgot about it, since there was no activity. If it's ok to keep skip complicated errors and move on without annotating it then we will get it done a lot faster. There is a lot of stuff to annotate.